portfolio/src/main/webapp/index.html

```diff
@@ -12,23 +12,168 @@
   </head>
   <body>
 
-    <div id="header"> 
+    <div class="loader" id="loaderid"></div>
+
+    <div class="hiddenclass" id="header"> 
         <div id="header-spacer"></div>
         <img class="headerels headerimages" src = "images/google.png"/>
-        <p class="headerels">Seun Omonije STEP Portfolio</p>
-        <input id="searchbar" class="headerels" type="text" placeholder="Search about me! (unimplemented)" />
+        <p class="headerels">STEP Portfolio</p>
+        <input id="searchbar" class="headerels" type="text" 
```

Comment:
Use oninput on this input box instead of onclick for the hidden submit button. This will run your function as users type (instead of on "enter"). You'll also be able to grab the contents of your search bar from the event instead of needing to query for the element.